src/modules/functions.js

```diff
@@ -0,0 +1,112 @@
+import removeIcon from '../img/remove.png';
+
+const icon3 = new Image();
+icon3.src = removeIcon;
+const todoList = [];
+
+export default class Todo {
+  constructor(todo) {
+    this.index = todoList.length + 1;
+    this.description = todo;
+    this.id = Date.now().toString();
+    this.completed = false;
+  }
+
+    static display = () => {
+      const todoItems = document.getElementById('id-list');
+      const todoList = JSON.parse(localStorage.getItem('todoList') || '[]');
+      let inputCheck = '';
+      todoItems.innerHTML = '';
+      todoList.forEach((task) => {
+        if (task.completed === false) {
+          inputCheck = '';
+        } else {
+          inputCheck = 'checked';
+        }
+        todoItems.innerHTML += `
+          <li>
+            <span class="item-info">
+              <input ${inputCheck} type="checkbox" class="check" id="input${task.index}">
+              <input id="${task.index}" class="task-item" value="${task.description}">
+            </span>
+            <span id="item-icon">
+              <img src="${icon3.src}" class="remove" id="${task.id}" alt="Remove Task" title="Remove Task">
+            </span>
+          </li>
+          `;
+      });
+    }
+/* eslint-disable */
```

Comment:
- [ ] It will be great to fix the linters error instead of disabling the linters. The linters will ensure you write great code. Kindly remove this statement and fix the linters.